neuraxle/base.py

```diff
@@ -115,6 +130,11 @@ def append(self, current_id, data_input, expected_output):
         self.data_inputs.append(data_input)
         self.expected_outputs.append(expected_output)
 
+    def concat(self, data_container: DataContainer):
+        self.current_ids.extend(data_container.current_ids)
+        self.data_inputs.extend(data_container.data_inputs)
+        self.expected_outputs.extend(data_container.expected_outputs)
```

Comment:
I repeat that you cannot call `.extend` on a Base DataContainer. The `data_inputs` and `expected_outputs` will most of the time not be of type `list`. However, a ListDataContainer would have this method. And a NumpyDataContainer could call what it needs by itself.

Reply:
it's already in the ListDataContainer... you didn't see the changes properly